api/PaymentAddress.json

```diff
@@ -92,67 +78,37 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/PaymentAddress/addressLine",
           "support": {
             "chrome": {
-              "version_added": "61",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "#web-payments",
-                  "value_to_set": "Enabled"
-                }
-              ]
+              "version_added": "60"
             },
             "chrome_android": {
-              "version_added": "53",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "#web-payments",
-                  "value_to_set": "Enabled"
-                }
-              ]
+              "version_added": "56"
             },
             "edge": {
               "version_added": "15"
             },
             "firefox": {
-              "version_added": "56",
-              "notes": "Available only in nightly builds. Requires the comma-delineated list in <code>dom.payments.request.supportedRegions</code> to contain one or more of the supported 2-character ISO locales, currently US and CA.",
-              "flags": [
```

Comment:
This flag removal I'm afraid isn't supported by the new guideline. Is this flag still there?

Reply:
Yeah, Firefox still requires a flag for the payment APIs.

Reply:
Good, so the data was correct. Can you revert any flag removals from this PR that are not supported by https://github.com/mdn/browser-compat-data/blob/master/docs/data-guidelines.md#removal-of-irrelevant-flag-data?

Reply:
Looking into this a little bit more, I did some research and confirmed the note about how this is only in Firefox Nightly builds.  No release build has support (even though the flags are available, they don't have any effect).  Would it perhaps make more sense to set Firefox to `false`?

Reply:
Oh, that's weird, I didn't know Firefox had compile-time features, or how this can happen. @ddbeck have you come across this situation before, and how should it be represented in BCD?

Reply:
We used to have a `compile_flag` but I removed it in #3752. #338 has some discussion about the need for a gated, non-release version option. Since we don't have anything like that yet, I'd suggest setting version_added to `false`, but keeping the note. (Better still would be to link from the note to the relevant bug so that, someday, when we have a solution to #338, updating it is easier.)

Reply:
Fixed!